.github/workflows/ci.yml

```diff
@@ -11,6 +11,10 @@ on:
 env:
   NXF_ANSI_LOG: false
 
+concurrency:
```

Comment:
What does this do?

Reply:
this is an nf-core change that I updated to